client/verta/verta/tracking/entities/_experimentrun.py

```diff
@@ -1720,7 +1724,7 @@ def download_artifact(self, key, download_to_path):
 
         # get a stream of the file bytes, without loading into memory, and write to file
         # TODO: consolidate this with _get_artifact() and get_artifact()
-        print("downloading {} from ModelDB".format(key))
+        logger.info("downloading %s from ModelDB", key)
```

Comment:
I'd generally be hesitant about doing piecemeal PR-by-PR transitions from `print` to `logger`, because some users 😬 _may_ expect to see print statements regarding downloads from using the client, and this breaks public-facing behavior (if they haven't configured their logging level).

That being said, the upcoming release will have backwards-incompatibilities anyway so this is fine.